test/test.js

```diff
@@ -0,0 +1,30 @@
+import chai from 'chai';
```

Comment:
@christianR15c , remember to include some pictures running these tests man.

Reply:
Ok, noted. I am going to add them